### PR TITLE
Make the send button and receive arrow occupy the same space

### PR DESCRIPTION
**Pull request by sakertooth**
This is a fix for another unintended change in #6591. Before, the send button and the receive arrow in the mixer channel view occupied the same space (i.e., either the send button was shown, or the receive arrow, without using two designated spots in the mixer channel for the two widgets, but one).

Here is a before and after. The difference is subtle, but notice how the receive arrow at the top was moved down a bit. This is because it is now occupying the same space the send button is occupying.

Before:
image

After:
image

### CODE REVIEW

File: include/MixerChannelView.h [hunk 1]
```diff
@@ -141,4 +137,4 @@ namespace lmms::gui
     };
 } // namespace lmms::gui
 
-#endif
\ No newline at end of file
+#endif
```
Comment (Rossmaxx) [inline include/MixerChannelView.h:140]:
```suggestion
#endif // MIXER_CHANNEL_VIEW_H
```
Not really necessary but would be nice.

**Review by Rossmaxx — APPROVED** [review]
Looks fine.

**Comment by sakertooth** [comment]
@Rossmaxx, do you notice any problem with rendering the effects in this PR?

**Comment by Rossmaxx** [comment]
> @Rossmaxx, do you notice any problem with rendering the effects in this PR?

Didn't check this PR but on latest master, there's a glitch. I didn't have this glitch on previous master, which was 9 commits behind. The only sus change seems to be #7502. I'll do something and check.

**Pull request merged**